tools/check_format.py

```diff
@@ -409,12 +419,19 @@ def checkFileContents(file_path, checker):
     # notes have a different format.
     checkCurrentReleaseNotes(file_path, error_messages)
 
+  format_flag = True
```

Comment:
Can this just use the `replaceLines` function above to make sure this logic is in a single place?

Reply:
We have a proposal incoming; the code becomes necessarily more convoluted to support a review vs. write-back scenario.